src/App/DefaultController.php

```diff
@@ -72,6 +72,9 @@ public function acceptableRequest(Request $request)
         return $mediaType;
     }
 
+    // TODO: remove? It's not in the api-raml, we are not using it for testing
+    // and it doesn't work anymore.
+    // SQLSTATE[42S22]: Column not found: 1054 Unknown column 'R.published' in 'order clause'
```

Comment:
Yes this was useful for mass-checking for bugs, so loading 100 results, logging exceptions and fixing anything that comes up when testing with the ~3000 articles. Should be able to remove now.